src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -25,7 +25,7 @@ const ProjectExternalSubmissionsTable = ({
   projectId: string;
   formDefinitionIdentifier: string;
 }) => {
-  const [selectedId, setSelectedId] = useState<string | null>(null);
+  const [modalOpenId, setModalOpenId] = useState<string | null>(null);
```

Comment:
Rename to avoid confusion over what "selected" means in this table. I'm also open to alternative names, maybe `viewingId`, `viewingSubmissionId`, etc.